src/libraries/System.Runtime.InteropServices.JavaScript/gen/JSImportGenerator/JSManagedTypeInfo.cs

```diff
@@ -163,6 +163,10 @@ public static JSTypeInfo CreateJSTypeInfoForTypeSymbol(ITypeSymbol type)
                         return new JSInvalidTypeInfo();
                     }
                     return new JSFunctionTypeInfo(false, signatureTypes);
+
+                // struct with inline array attribute
+                case { TypeKind: TypeKind.Struct } structType when structType.GetAttributes().Any(attr => attr.AttributeClass?.ToDisplayString() == "System.Runtime.CompilerServices.InlineArrayAttribute"):
```

Comment:
Why is it needed if it's same as the fallback?

Reply:
Good point. Currently, the JS Interop generator does not support the marshalling of structs. This was done to allow for potential future extensibility, to disallow marshalling of structs with the ` InlineArrayAttribute` in case structs were to be allowed for marshalling in the future.

However, if it is not necessary, we can leave a comment as a reminder.